pyfar/dsp/dsp.py

```diff
@@ -143,6 +143,51 @@ def wrap_to_2pi(x):
     return x
 
 
+def linear_phase(signal, group_delay=None):
```

Comment:
I am not a fan of having a group delay which is None by default, this is potentially confusing implicit behavior

Reply:
I'm not sure if this works, if yes it can be `group_delay=signal.n_samples/2`